LPTIM Kconfig-related build failures for nucleo_g431rb

**Describe the bug**
Multiple tests fail to build for `nucleo_g431rb` with LPTIM Kconfig deprecation-related warnings:

```
tests/benchmarks/footprints/benchmark.kernel.footprints.pm
tests/kernel/profiling/profiling_api/kernel.common.profiling
tests/kernel/tickless/tickless_concept/kernel.tickless.concept
tests/kernel/timer/timer_api/kernel.timer.tickless
tests/subsys/pm/device_runtime_api/pm.device_runtime.api
```

**To Reproduce**
```
scripts/twister -v -N -M -p nucleo_g431rb -s tests/benchmarks/footprints/benchmark.kernel.footprints.pm
```

**Expected behavior**
Tests build without any deprecation warnings.

**Impact**
Use of deprecated features

**Logs and console output**
```
Loading Zephyr default modules (Zephyr base).
-- Application: /__w/zephyr/zephyr/tests/subsys/pm/device_runtime_api
-- CMake version: 3.20.5
-- Found Python3: /usr/bin/python3.8 (found suitable exact version "3.8.10") found components: Interpreter 
-- Cache files will be written to: /__w/zephyr/zephyr/.cache
-- Zephyr version: 3.3.0-rc1 (/__w/zephyr/zephyr)
-- Found west (found suitable version "0.14.0", minimum required is "0.7.1")
-- Board: nucleo_g431rb
-- Found host-tools: zephyr 0.15.2 (/opt/toolchains/zephyr-sdk-0.15.2)
-- Found toolchain: zephyr 0.15.2 (/opt/toolchains/zephyr-sdk-0.15.2)
-- Found Dtc: /opt/toolchains/zephyr-sdk-0.15.2/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.6.0", minimum required is "1.4.6") 
-- Found BOARD.dts: /__w/zephyr/zephyr/boards/arm/nucleo_g431rb/nucleo_g431rb.dts
-- Generated zephyr.dts: /__w/zephyr/zephyr/twister-out/nucleo_g431rb/tests/subsys/pm/device_runtime_api/pm.device_runtime.api/zephyr/zephyr.dts
-- Generated devicetree_generated.h: /__w/zephyr/zephyr/twister-out/nucleo_g431rb/tests/subsys/pm/device_runtime_api/pm.device_runtime.api/zephyr/include/generated/devicetree_generated.h
-- Including generated dts.cmake file: /__w/zephyr/zephyr/twister-out/nucleo_g431rb/tests/subsys/pm/device_runtime_api/pm.device_runtime.api/zephyr/dts.cmake
/__w/zephyr/zephyr/twister-out/nucleo_g431rb/tests/subsys/pm/device_runtime_api/pm.device_runtime.api/zephyr/zephyr.dts:408.7-431.5: Warning (simple_bus_reg): /soc/can: missing or empty reg/ranges property
Parsing /__w/zephyr/zephyr/Kconfig
Loaded configuration '/__w/zephyr/zephyr/boards/arm/nucleo_g431rb/nucleo_g431rb_defconfig'
Merged configuration '/__w/zephyr/zephyr/tests/subsys/pm/device_runtime_api/prj.conf'
Configuration saved to '/__w/zephyr/zephyr/twister-out/nucleo_g431rb/tests/subsys/pm/device_runtime_api/pm.device_runtime.api/zephyr/.config'
Kconfig header saved to '/__w/zephyr/zephyr/twister-out/nucleo_g431rb/tests/subsys/pm/device_runtime_api/pm.device_runtime.api/zephyr/include/generated/autoconf.h'
-- The C compiler identification is GNU 12.1.0
-- The CXX compiler identification is GNU 12.1.0
-- The ASM compiler identification is GNU
-- Found assembler: /opt/toolchains/zephyr-sdk-0.15.2/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc
-- Configuring done
-- Generating done
-- Build files have been written to: /__w/zephyr/zephyr/twister-out/nucleo_g431rb/tests/subsys/pm/device_runtime_api/pm.device_runtime.api
[1/170] Preparing syscall dependency handling

[2/170] Generating include/generated/version.h
-- Zephyr version: 3.3.0-rc1 (/__w/zephyr/zephyr), build: v3.3.0-rc1-97-g432ff20a72e1
[3/170] Generating misc/generated/syscalls_subdirs.trigger
[4/170] Generating misc/generated/syscalls.json, misc/generated/struct_tags.json
[5/170] Generating include/generated/syscall_dispatch.c, include/generated/syscall_list.h
[6/170] Generating include/generated/driver-validation.h
[7/170] Generating include/generated/kobj-types-enum.h, include/generated/otype-to-str.h, include/generated/otype-to-size.h
[8/170] Building C object zephyr/CMakeFiles/offsets.dir/arch/arm/core/offsets/offsets.c.obj
[9/170] Generating include/generated/offsets.h
[10/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/dec.c.obj
[11/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/hex.c.obj
[12/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc32_sw.c.obj
[13/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/multi_heap.c.obj
[14/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc8_sw.c.obj
[15/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc32c_sw.c.obj
[16/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc16_sw.c.obj
[17/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/timeutil.c.obj
[18/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/crc7_sw.c.obj
[19/170] Building C object zephyr/CMakeFiles/zephyr.dir/misc/generated/configs.c.obj
[20/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/fdtable.c.obj
[21/170] Building C object CMakeFiles/app.dir/src/test_driver.c.obj
[22/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/thread_entry.c.obj
[23/170] Building C object CMakeFiles/app.dir/src/main.c.obj
[24/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/sem.c.obj
[25/170] Linking C static library app/libapp.a
[26/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/printk.c.obj
[27/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/cbprintf_packaged.c.obj
[28/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/assert.c.obj
[29/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/cbprintf.c.obj
[30/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/rb.c.obj
[31/170] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/tracing/tracing_none.c.obj
[32/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/heap-validate.c.obj
[33/170] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/pm/state.c.obj
[34/170] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/common/soc_config.c.obj
[35/170] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/stm32g4/soc.c.obj
[36/170] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/common/stm32cube_hal.c.obj
[37/170] Generating linker_zephyr_pre1.cmd
[38/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/heap.c.obj
[39/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/bitarray.c.obj
[40/170] Generating linker_zephyr_pre0.cmd
[41/170] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/cpu_idle.S.obj
[42/170] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/nmi_on_reset.S.obj
[43/170] Building C object zephyr/CMakeFiles/zephyr.dir/soc/arm/st_stm32/stm32g4/power.c.obj
[44/170] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/logging/log_minimal.c.obj
[45/170] Building C object zephyr/CMakeFiles/zephyr.dir/lib/os/cbprintf_complete.c.obj
[46/170] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/isr_wrapper.S.obj
[47/170] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/fault_s.S.obj
[48/170] Building ASM object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/swap_helper.S.obj
[49/170] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/exc_exit.S.obj
[50/170] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/pm/pm.c.obj
[51/170] Building C object zephyr/arch/common/CMakeFiles/isr_tables.dir/isr_tables.c.obj
[52/170] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/pm/policy.c.obj
[53/170] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/pm/device.c.obj
[54/170] Building C object zephyr/CMakeFiles/zephyr.dir/subsys/pm/device_runtime.c.obj
[55/170] Linking C static library zephyr/arch/common/libisr_tables.a
[56/170] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/reset.S.obj
[57/170] Building ASM object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/vector_table.S.obj
[58/170] Linking C static library zephyr/libzephyr.a
[59/170] Generating ../../../include/generated/libc/minimal/strerror_table.h
[60/170] Building C object zephyr/arch/common/CMakeFiles/arch__common.dir/sw_isr_common.c.obj
[61/170] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/swap.c.obj
[62/170] Linking C static library zephyr/arch/common/libarch__common.a
[63/170] Building C object zephyr/CMakeFiles/zephyr_pre0.dir/misc/empty_file.c.obj
[64/170] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/nmi.c.obj
[65/170] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/prep_c.c.obj
[66/170] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/thread.c.obj
[67/170] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/fatal.c.obj
[68/170] Building C object zephyr/arch/arch/arm/core/aarch32/CMakeFiles/arch__arm__core__aarch32.dir/irq_manage.c.obj
[69/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/atoi.c.obj
[70/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtol.c.obj
[71/170] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/fpu.c.obj
[72/170] Linking C static library zephyr/arch/arch/arm/core/aarch32/libarch__arm__core__aarch32.a
[73/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtoul.c.obj
[74/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtoll.c.obj
[75/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/bsearch.c.obj
[76/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/strtoull.c.obj
[77/170] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/irq_init.c.obj
[78/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strstr.c.obj
[79/170] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/scb.c.obj
[80/170] Building C object zephyr/soc/arm/common/cortex_m/CMakeFiles/soc__arm__common__cortex_m.dir/arm_mpu_regions.c.obj
[81/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strerror.c.obj
[82/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/qsort.c.obj
[83/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strspn.c.obj
[84/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/strncasecmp.c.obj
[85/170] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/thread_abort.c.obj
[86/170] Linking C static library zephyr/soc/arm/common/cortex_m/libsoc__arm__common__cortex_m.a
[87/170] Building C object zephyr/arch/arch/arm/core/aarch32/mpu/CMakeFiles/arch__arm__core__aarch32__mpu.dir/arm_core_mpu.c.obj
[88/170] Building C object zephyr/subsys/testsuite/ztest/CMakeFiles/subsys__testsuite__ztest.dir/src/ztest_error_hook.c.obj
[89/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/math/sqrtf.c.obj
[90/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/math/sqrt.c.obj
[91/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/string/string.c.obj
[92/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/abort.c.obj
[93/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/exit.c.obj
[94/170] Building C object zephyr/arch/arch/arm/core/aarch32/mpu/CMakeFiles/arch__arm__core__aarch32__mpu.dir/arm_mpu.c.obj
[95/170] Building C object zephyr/subsys/testsuite/ztest/CMakeFiles/subsys__testsuite__ztest.dir/src/ztest_defaults.c.obj
[96/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdlib/malloc.c.obj
[97/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/fprintf.c.obj
[98/170] Linking C static library zephyr/arch/arch/arm/core/aarch32/mpu/libarch__arm__core__aarch32__mpu.a
[99/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/sprintf.c.obj
[100/170] Building C object zephyr/subsys/testsuite/ztest/CMakeFiles/subsys__testsuite__ztest.dir/src/ztest_new.c.obj
[101/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/time/gmtime.c.obj
[102/170] Building C object zephyr/subsys/testsuite/ztest/CMakeFiles/subsys__testsuite__ztest.dir/src/ztest_rules.c.obj
[103/170] Building C object zephyr/drivers/clock_control/CMakeFiles/drivers__clock_control.dir/clock_stm32g4.c.obj
[104/170] Building C object zephyr/lib/libc/minimal/CMakeFiles/lib__libc__minimal.dir/source/stdout/stdout_console.c.obj
[105/170] Linking C static library zephyr/subsys/testsuite/ztest/libsubsys__testsuite__ztest.a
[106/170] Building C object zephyr/drivers/timer/CMakeFiles/drivers__timer.dir/sys_clock_init.c.obj
[107/170] Building C object zephyr/drivers/clock_control/CMakeFiles/drivers__clock_control.dir/clock_stm32_ll_common.c.obj
[108/170] Building C object zephyr/drivers/console/CMakeFiles/drivers__console.dir/uart_console.c.obj
[109/170] Linking C static library zephyr/lib/libc/minimal/liblib__libc__minimal.a
[110/170] Building C object zephyr/drivers/pinctrl/CMakeFiles/drivers__pinctrl.dir/common.c.obj
[111/170] Linking C static library zephyr/drivers/clock_control/libdrivers__clock_control.a
[112/170] Building C object zephyr/arch/arch/arm/core/aarch32/cortex_m/CMakeFiles/arch__arm__core__aarch32__cortex_m.dir/fault.c.obj
[113/170] Linking C static library zephyr/drivers/console/libdrivers__console.a
[114/170] Building C object zephyr/drivers/interrupt_controller/CMakeFiles/drivers__interrupt_controller.dir/intc_exti_stm32.c.obj
[115/170] Linking C static library zephyr/arch/arch/arm/core/aarch32/cortex_m/libarch__arm__core__aarch32__cortex_m.a
[116/170] Linking C static library zephyr/drivers/interrupt_controller/libdrivers__interrupt_controller.a
[117/170] Building C object zephyr/drivers/gpio/CMakeFiles/drivers__gpio.dir/gpio_stm32.c.obj
[118/170] Building C object zephyr/drivers/pinctrl/CMakeFiles/drivers__pinctrl.dir/pinctrl_stm32.c.obj
[119/170] Building C object zephyr/drivers/reset/CMakeFiles/drivers__reset.dir/reset_stm32.c.obj
[120/170] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32g4xx/soc/system_stm32g4xx.c.obj
[121/170] Linking C static library zephyr/drivers/gpio/libdrivers__gpio.a
[122/170] Linking C static library zephyr/drivers/reset/libdrivers__reset.a
[123/170] Linking C static library zephyr/drivers/pinctrl/libdrivers__pinctrl.a
[124/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/device.c.obj
[125/170] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32g4xx/drivers/src/stm32g4xx_hal.c.obj
[126/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/version.c.obj
[127/170] Building C object zephyr/drivers/timer/CMakeFiles/drivers__timer.dir/stm32_lptim_timer.c.obj
FAILED: zephyr/drivers/timer/CMakeFiles/drivers__timer.dir/stm32_lptim_timer.c.obj 
ccache /opt/toolchains/zephyr-sdk-0.15.2/arm-zephyr-eabi/bin/arm-zephyr-eabi-gcc -DCORE_CM4 -DHSE_VALUE=24000000 -DKERNEL -DSTM32G431xx -DTC_RUNID=c8ef868b856b4671422acd793b6a2990 -DUSE_FULL_LL_DRIVER -DUSE_HAL_DRIVER -D__PROGRAM_START -D__ZEPHYR_SUPERVISOR__ -D__ZEPHYR__=1 -I../../../../../../../include -Izephyr/include/generated -I../../../../../../../soc/arm/st_stm32/stm32g4 -I../../../../../../../drivers -I../../../../../../../soc/arm/st_stm32/common -I../../../../../../../subsys/testsuite/include -I../../../../../../../subsys/testsuite/ztest/include -I/__w/zephyr/modules/hal/cmsis/CMSIS/Core/Include -I/__w/zephyr/modules/hal/stm32/stm32cube/stm32g4xx/soc -I/__w/zephyr/modules/hal/stm32/stm32cube/stm32g4xx/drivers/include -I/__w/zephyr/modules/hal/stm32/stm32cube/stm32g4xx/drivers/include/Legacy -I/__w/zephyr/modules/hal/stm32/stm32cube/common_ll/include -isystem ../../../../../../../lib/libc/minimal/include -isystem /opt/toolchains/zephyr-sdk-0.15.2/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.1.0/include -isystem /opt/toolchains/zephyr-sdk-0.15.2/arm-zephyr-eabi/bin/../lib/gcc/arm-zephyr-eabi/12.1.0/include-fixed -fno-strict-aliasing -Os -imacros /__w/zephyr/zephyr/twister-out/nucleo_g431rb/tests/subsys/pm/device_runtime_api/pm.device_runtime.api/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -mfp16-format=ieee --sysroot=/opt/toolchains/zephyr-sdk-0.15.2/arm-zephyr-eabi/arm-zephyr-eabi -imacros /__w/zephyr/zephyr/include/zephyr/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-unused-but-set-variable -Werror=implicit-int -fno-pic -fno-pie -Werror -fno-asynchronous-unwind-tables -fno-reorder-functions --param=min-pagesize=0 -fno-defer-pop -fmacro-prefix-map=/__w/zephyr/zephyr/tests/subsys/pm/device_runtime_api=CMAKE_SOURCE_DIR -fmacro-prefix-map=/__w/zephyr/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/__w/zephyr=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT zephyr/drivers/timer/CMakeFiles/drivers__timer.dir/stm32_lptim_timer.c.obj -MF zephyr/drivers/timer/CMakeFiles/drivers__timer.dir/stm32_lptim_timer.c.obj.d -o zephyr/drivers/timer/CMakeFiles/drivers__timer.dir/stm32_lptim_timer.c.obj -c /__w/zephyr/zephyr/drivers/timer/stm32_lptim_timer.c
/__w/zephyr/zephyr/drivers/timer/stm32_lptim_timer.c:32:2: error: #warning Kconfig for LPTIM source clock (LSI/LSE) is deprecated, use device tree. [-Werror=cpp]
   32 | #warning Kconfig for LPTIM source clock (LSI/LSE) is deprecated, use device tree.
      |  ^~~~~~~
cc1: all warnings being treated as errors
[128/170] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32g4xx/drivers/src/stm32g4xx_hal_rcc.c.obj
[129/170] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32g4xx/drivers/src/stm32g4xx_ll_utils.c.obj
[130/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/main_weak.c.obj
[131/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/banner.c.obj
[132/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/errno.c.obj
[133/170] Building C object modules/stm32/stm32cube/CMakeFiles/..__modules__hal__stm32__stm32cube.dir/stm32g4xx/drivers/src/stm32g4xx_hal_rcc_ex.c.obj
[134/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/init.c.obj
[135/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/idle.c.obj
[136/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mem_slab.c.obj
[137/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/kheap.c.obj
[138/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/fatal.c.obj
[139/170] Building C object zephyr/drivers/serial/CMakeFiles/drivers__serial.dir/uart_stm32.c.obj
[140/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/msg_q.c.obj
[141/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mailbox.c.obj
[142/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/queue.c.obj
[143/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/thread.c.obj
[144/170] Building C object zephyr/kernel/CMakeFiles/kernel.dir/mutex.c.obj
ninja: build stopped: subcommand failed.
```

**Environment (please complete the following information):**
 - OS: Ubuntu 20.04
 - Toolchain: Zephyr SDK 0.15.2
 - Commit SHA: https://github.com/zephyrproject-rtos/zephyr/commit/432ff20a72e1bfbc95b40c589b55075d4d82f22e

--- a/boards/arm/nucleo_g431rb/Kconfig.defconfig
+++ b/boards/arm/nucleo_g431rb/Kconfig.defconfig
@@ -12,4 +12,8 @@ config SPI_STM32_INTERRUPT
 	default y
 	depends on SPI
 
+# LPTIM clocked by LSE, force tick freq to 4096 for tick accuracy
+config SYS_CLOCK_TICKS_PER_SEC
+	default 4096 if STM32_LPTIM_TIMER
+
 endif # BOARD_NUCLEO_G431RB

--- a/boards/arm/nucleo_g431rb/nucleo_g431rb.dts
+++ b/boards/arm/nucleo_g431rb/nucleo_g431rb.dts
@@ -56,6 +56,10 @@
 	status = "okay";
 };
 
+&clk_lse {
+	status = "okay";
+};
+
 &clk_hse {
 	clock-frequency = <DT_FREQ_M(24)>;
 	status = "okay";
@@ -129,6 +133,12 @@
 		pinctrl-0 = <&tim2_ch1_pa5>;
 		pinctrl-names = "default";
 	};
+};
+
+&lptim1 {
+	clocks = <&rcc STM32_CLOCK_BUS_APB1 0x80000000>,
+		 <&rcc STM32_SRC_LSE LPTIM1_SEL(3)>;
+	status = "okay";
 };
 
 &rtc {

